frontend/package.json

```diff
@@ -14,6 +18,7 @@
     "react-dom": "^16.13.1",
     "react-helmet": "^5.2.1",
     "react-redux": "^7.2.0",
+    "react-router-dom": "^5.1.2",
```

Comment:
Good point! I actually wanted to talk to you about the router package. For the pickup signup process we will need to pass along the information that the user enter from one page to the next. Do you know how to do this with `@reach/router`/ if it's possible? 

I'm pretty sure there is a way to do this with `react-router-dom` but since we are using `@reach/router` wanted to see if there is a way of doing it with this library?

Reply:
For passing information from one page to the next I would use Redux rather than routing params, but if we also need URL parameters (like for activation links) [its pretty easy to accomplish this](https://reach.tech/router/example/url-params) in almost the same way you would with `react-router-dom`

Reply:
Agreed, redux would make a lot of things easier actually and make our code cleaner ... especially once we get the endpoints.